### PR TITLE
feat(folderReader): add constants for events call

### PR DESCRIPTION
**Pull request by Teyz**
In this PR : 
- Adding constants file for events call

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { app, protocol, BrowserWindow, ipcMain } from "electron";
 import { createProtocol } from "vue-cli-plugin-electron-builder/lib";
 import installExtension, { VUEJS3_DEVTOOLS } from "electron-devtools-installer";
 import FolderReader from "./services/folderReader";
+import constants from "../utils/constants";
```
Comment (felixbouveret) [inline src/background.js:7]:
```suggestion
import { GET_FOLDER_PATH_EVENT, GET_FOLDER_PATH_REPLY } from "../utils/constants";
```
So you don't need to add `constants.CONST` everywhere

File: src/services/folderReader.js [hunk 2]
```diff
@@ -25,10 +25,13 @@ class folderReader {
   checkIsGitFolder(folderPath) {
     return access(`${folderPath}/.git`)
       .then(() => {
-        return {folderPath, isGit: true};
+        return { folderPath, isGit: true };
       })
```
Comment (felixbouveret) [inline src/services/folderReader.js:29]: could be replaced by 
``` suggestion
      .then(() => ({ folderPath, isGit: true }))
```

File: src/services/folderReader.js [hunk 3]
```diff
@@ -25,10 +25,13 @@ class folderReader {
...
       .catch(() => {
-        return {error : "This folder does not contain a git folder", isGit: false};
+        return {
+          error: "This folder does not contain a git folder",
+          isGit: false,
+        };
       });
```
Comment (felixbouveret) [inline src/services/folderReader.js:35]: Same here

**Comment by felixbouveret** [comment]
@Teyz is making a lot of bêtises that I have répared

**Review by antoinechassagne — APPROVED** [review]
👍

**Pull request merged**